### ISSUE TITLE
Any chance of v4 compatibility?

### ISSUE THREAD

**Issue by danwdart** [airbrake/airbrake-wordpress#3]
Having to use Wordpress (at least I get to use the newest) but this plugin doesn't seem to have been checked for that.

Any chance of an upgrade check?
Cheers

**Comment by kyrylo** [comment]
Hi! Thanks for filing this issue :+1:

I think we will consider that in the near future.

**Comment by md5** [comment]
I was going to open a new issue to ask if there are any plans to update this plugin, but it's similar enough to this issue that I'll piggy-back.

I just saw that `phpbrake` 0.1.0 was released today, so it seems like it would be cool if this plugin used that library.

In my case, I'm looking to log from WordPress to Errbit, so I'd love to have the ability to configure the reporting host. My understanding is that this would help Airbrake Custom customers as well.
